cyber-resilience-sig/whitepapers/stewards-and-cra.md

```diff
@@ -13,23 +13,386 @@ License: CC-BY 4.0
+- the highest number of users of products of that Steward are located
+
+In practice, we can assume that the Steward/Project need to report by default to the CSIRT of the country the Steward has its headquaters.
+If the Steward has no European official representation, we assume the reporting can be done to one of the CSIRTs of big EU countries (like Germany, France, Spain, Italy etc),
+except is the specific Project has a strong user base in another country.
+
+Related ORC FAQ entries: [on choice of CSIRT](https://github.com/orcwg/cra-hub/issues/167)
+
+References: 24(3), Directive 2022/2555
+
+        Directive 2022/2555, Article 12(1) "Coordinated vulnerability disclosure and a European vulnerability database":
+
+        Each Member State shall designate one of its CSIRTs as a coordinator for the purposes of coordinated vulnerability disclosure.
+        The CSIRT designated as coordinator shall act as a trusted intermediary, facilitating, where necessary, the interaction between
+        the natural or legal person reporting a vulnerability and the manufacturer or provider of the potentially vulnerable ICT products
+        or ICT services, upon the request of either party. The tasks of the CSIRT designated as coordinator shall include:
+        (a) identifying and contacting the entities concerned;
+        (b) assisting the natural or legal persons reporting a vulnerability; and
+        (c) negotiating disclosure timelines and managing vulnerabilities that affect multiple entities.
+
+        Member States shall ensure that natural or legal persons are able to report, anonymously where they so request, a vulnerability
+        to the CSIRT designated as coordinator. The CSIRT designated as coordinator shall ensure that diligent follow-up action is carried
+        out with regard to the reported vulnerability and shall ensure the anonymity of the natural or legal person reporting the vulnerability.
+        Where a reported vulnerability could have a significant impact on entities in more than one Member State, the CSIRT designated as
+        coordinator of each Member State concerned shall, where appropriate, cooperate with other CSIRTs designated as coordinators
+        within the CSIRTs network.
+
+#### Market surveillance authorities
+
+Market surveillance authorities might request information from the Steward on their policies.
+
+Steward must know who is their market surveillance authority. The same rules apply as for the choice of CSIRT.
+        
+## Restrictions
+
+As Stewards (and open source projects) follow less strict rules than PwDE placed on the market by Manufacturers, Steward cannot affix
+the CE marking on the products they publish. It also means that they do not provide the complete CRA documentations for its Projects.
+
+They might have an attestation program that provides equivalent documentation to the use of manufacturers.
```

Comment:
"... to the use ..." -> "... for the use ..."

Reply:
Fixed thanks!